test/langtools/tools/javac/patterns/NestedPrimitiveDeconstructionPattern.java

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Nit - maybe add the empty line behind or before the `@test` comment, just to be sure header checkers are not confused?

Reply:
Okay -- must have accidentally added a blank line.